Dockerfile

```diff
@@ -21,22 +21,20 @@ RUN echo 'deb https://download.opensuse.org/repositories/devel:/tools:/criu/Debi
         curl \
         gawk \
         gcc \
+        gperf \
         iptables \
         jq \
         kmod \
-        libseccomp-dev \
-        libseccomp-dev:arm64 \
-        libseccomp-dev:armel \
-        libseccomp-dev:armhf \
-        libseccomp-dev:ppc64el \
-        libseccomp2 \
         pkg-config \
         python-minimal \
         sudo \
         uidmap \
     && apt-get clean \
     && rm -rf /var/cache/apt /var/lib/apt/lists/* /etc/apt/sources.list.d/*.list
 
+COPY script/install-libseccomp.sh /
+RUN /install-libseccomp.sh
```

Comment:
We will be able to simplify this when:
 * debian bullseye is released, estimated for Aug 14 (date has been moving, not sure if that is expected again or not) 
 * _And_ if golang releases a new image using bullseye as base

In that case, we can just revert this chunk and update to bullseye. But not sure if there are other things to consider when changing the dockerfile to bullseye.

I can add a comment with that here, so we don't forget

Reply:
Added a comment for this in the same commit, next to the `FROM` line that currently uses buster

Reply:
Debian Bullseye is now available -- see https://github.com/opencontainers/runc/pull/3171 -- so maybe it's better to use it.

Reply:
Ok, chery-picked your commit here!